compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -322,4 +322,10 @@ def VerifyInputLegality: Pass<"iree-verify-input-legality", ""> {
   let constructor = "mlir::iree_compiler::IREE::Flow::createVerifyInputLegalityPass()";
 }
 
+def DequantizationMatmulFuse:
```

Comment:
Similarly, could you name this as `FuseDequantizationMatmul`? Also the pass in the C++ file.